src/control/common/storage_types.go

```diff
@@ -38,13 +38,13 @@ func (nc NvmeControllers) String() string {
 	var buf bytes.Buffer
 
 	for _, ctrlr := range nc {
-		fmt.Fprintf(
-			&buf, "\tPCI Address:%s Serial:%s\n\tModel:%s Fwrev:%s\n",
-			ctrlr.Pciaddr, ctrlr.Serial, ctrlr.Model, ctrlr.Fwrev)
+		fmt.Fprintf(&buf,
+			"\tPCI Address:%s Serial:%s Model:%s Fwrev:%s Socket: %d\n",
```

Comment:
One concern I have with this formatting choice is that the line break makes it harder to run the output through grep. I'd recommend removing any mid-entry line breaks, even if it results in longer output lines.

Reply:
agreed, presumably the json option could be used for that